### PR TITLE
ci: Add github CI

### PR DESCRIPTION
**Pull request by pythonbrad**

### CODE REVIEW

File: .github/workflows/pylint.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    - uses: actions/checkout@v3
+    - name: Set up Python
+      uses: actions/setup-python@v3
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install pylint
+    - name: Analysing the code with pylint
+      run: |
+        pylint $(git ls-files '*.py')
```
Comment (Sanix-Darker) [inline .github/workflows/pylint.yaml:18]: is it possible to use the integrated linter from the project ? there is ruff, black and mypy, why using pylint here ? We should be sync from our dev env and the CI env right ? 🤔

Comment (pythonbrad) [reply .github/workflows/pylint.yaml:18]: 😯 , i hadn't forgotten it.

Comment (Sanix-Darker) [reply .github/workflows/pylint.yaml:18]: https://github.com/dylantientcheu/lamap-bot/blob/v2/Makefile#L18

Comment (pythonbrad) [reply .github/workflows/pylint.yaml]: I updated

**Review by Sanix-Darker — APPROVED** [review]
lgtm

**Pull request merged**